content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
 
 ### Using the bundle {#create-simulation-bundle}
 
-#### Step 3.1 - Generate a Package {#generate-package}
+#### Step 3.1 - Create an API token
+
+On Gatling Enterprise, click on the [API Tokens section]({{< ref "../../reference/admin/api_tokens/" >}}) and on create. Give a name to your API token, and give him `Configure` as Organization role, then hit save.
 
-Gatling Enterprise requires a package built from a Gatling project in order to run a test.
+Next, copy the content of the API token, you won't be able to access it once you close the modal.
 
-To create a package, run the `enterprisePackage.bat` (if Windows) or `enterprisePackage.sh` (if Linux/MacOS) file, Gatling will start creating your first package:
+#### Step 3.2 - Create and run your first simulation on the cloud
+
+We will use the CLI tool interact with `cloud.gatling.io`, so run it with the API token you just generated : 
 
 ```
-$ ./bin/enterprisePackage.sh
-GATLING_HOME is set to .../gatling-charts-highcharts-bundle-{{< var gatlingVersion >}}
-GATLING_VERSION is set to '{{< var gatlingVersion >}}'
-Creating jar...done
+$ ./bin/gatling.sh --api-token <API_TOKEN>
 ```
 
-This script can also be clicked or double clicked from any file explorer. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
-
-{{< youtube _fq3giAIibw >}}
+```
+Creating the package
+Package created
+Do you want to create a new simulation or start an existing one?
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Create a new Simulation on Gatling Enterprise, then start it
+[2] Start an existing Simulation on Gatling Enterprise
+```
 
-#### Step 3.2 - Upload the Package {#upload-package}
+Let's create a new simulation, type `1`
 
-On Gatling Enterprise, click on the [Packages section]({{< ref "../../reference/user/package_conf/" >}}) and on create. Give a name to your package, you can choose to make it available to all teams or to limit access to a specific one, then hit save.
+```
+Proceeding to the create simulation step
+Choose one simulation class in your package:
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] computerdatabase.ComputerDatabaseSimulation
+```
 
-Next, upload the package you generated earlier by clicking on the cloud icon. The package you generated earlier is located here: `target/package.jar`
+Select your Simulation class (the only available here) with `1`
 
-Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+```
+Choose a team from the list:
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Team 'Default team', id='b236af88-1c24-4623-b872-cb8993651c15'
+```
 
-#### Step 3.3 - Create a Simulation {#create-simulation}
+Select a special team, or the default one for your organization with `1`. This will allow access to the simulation to users within this team.
```

Comment:
Remove the "Select a special team", the getting started should be used with a new organization, with only the default team